courses/machine_learning/deepdive2/text_classification/labs/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>I'm torn between the flexibility this lab offers and the potentially overwhelming number of BERT and related model variants available. Since ASL labs are generally targeting 45 min - 1 hour, maybe consider being more perspective? Some options here are to to reduce set below and keep the links you have here. Others might be just setting up the small bert example first and then having a continue your learning section below that has the rest of the models for students to experiment with. Shop this around with the team for alternative opinions for sure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1412/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Thanks for this suggestion! I decided to add a continued learning section at the end for the curious students for now.</p>